### PR TITLE
[RDY] Windows fixes

### PR DESCRIPTION
**Pull request by b-r-o-c-k**

### CODE REVIEW

**Comment by justinmk** [comment]
@leonerd long-term patch: dc06fb2

**Comment by justinmk** [comment]
Looks like libtermkey has the same issue, from AppVeyor logs:

```
[00:04:20] c:\projects\neovim\.deps\build\src\libtermkey\t\39dcs.c(15): warning C4129: 'e': unrecognized character escape sequence [C:\projects\neovim\.deps\build\src\libtermkey-build\test_39dcs.vcxproj] [C:\projects\neovim\.deps\libtermkey.vcxproj]
```

cc @leonerd

**Comment by justinmk** [comment]
@erw7 Does this resolve anything related to https://github.com/neovim/neovim/pull/8462 ?

**Comment by erw7** [comment]
@justinmk  thanks. But this seems to solve the problem related to the escape sequence in MSVC. I think this is unrelated to # 8462.

File: src/nvim/tui/tui.c [hunk 1]
```diff
@@ -845,7 +845,7 @@ CursorShape tui_cursor_decode_shape(const char *shape_str)
 
 static cursorentry_T decode_cursor_entry(Dictionary args)
 {
-  cursorentry_T r;
+  cursorentry_T r = { 0 };
```
Comment (justinmk) [inline src/nvim/tui/tui.c]: My suggestion here wasn't good, it fails quickbuild CI due to a (valid) warning. Maybe this instead:

    cursorentry_T r = shape_table[0];

**Comment by justinmk** [comment]
Merged, thanks!

**Pull request closed without merging**